webpack.test.config.js

```diff
@@ -1,65 +1,12 @@
-var webpack = require( 'webpack' ),
+const webpack = require( 'webpack' ),
 	path = require( 'path' ),
-	nodeExternals = require( 'webpack-node-externals' );
+	nodeExternals = require( 'webpack-node-externals' )
+	webpackConfig = require( './webpack.config' );
 
-module.exports = {
-	cache: true,
-	module: {
-		loaders: [
-			{
-				test: /\.json$/,
-				loader: 'json-loader'
-			},
-			{
-				test: /\.html$/,
-				loader: 'html-loader'
-			},
-			{
-				test: /\.scss$/,
-				loader: 'noop-loader',
-			},
-			{
-				test: /\.jsx?$/,
-				loader: 'babel-loader',
-				query: {
-					cacheDirectory: true,
-					presets: [
-						'es2015',
-						'stage-1',
-						'react'
-					],
-					plugins: [
-						"add-module-exports",
-					],
-					babelrc: false,
-				}
-			}
-		]
-	},
-	resolve: {
-		alias: {
-			'react': path.join( __dirname, 'node_modules', 'react' ),
-			'react-dom': path.join( __dirname, 'node_modules', 'react-dom' ),
-			'redux': path.join( __dirname, 'node_modules', 'redux' ),
-			'lib/mixins/i18n': path.join( __dirname, 'client', 'lib', 'mixins', 'i18n' )
-		},
-		extensions: [ '', '.json', '.js', '.jsx' ],
-		root: [
-			path.join( __dirname, 'client' ),
-			path.join( __dirname, 'node_modules' ),
-			path.join( __dirname, 'node_modules', 'wp-calypso', 'client' )
-		],
-		fallback: [
-			path.join( __dirname, 'node_modules', 'wp-calypso', 'node_modules' )
-		]
-	},
-	resolveLoader: {
-		modulesDirectories: [ __dirname + '/node_modules' ]
-	},
-	target: 'node',
-	externals: [
-		nodeExternals( {
-			whitelist: [ 'wp-calypso' ]
-		} )
-	]
-};
\ No newline at end of file
+webpackConfig.externals = [
```

Comment:
I am not 100% sure that's true, but even if it was, what kind of problems do you think will produce? This `webpack.test.config.js` is only loaded for tests, so it won't interfere with the normal course of operations. Can you produce an issue with doing this?